### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SmartSergiZ**
[DEMO LINK](https://SmartSergiZ.github.io/layout_catalog/)
[TEST REPORT LINK](https://SmartSergiZ.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      padding-right: 4px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      width: 16px;
+      height: 16px;
+    }
+  }
+
+  &__stars--is-active > :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```
Comment (alexander-ignatow) [inline src/styles/blocks/card.scss:89]: You were supposed to reuse the stars block from the old stars task

For now, it is no easy way to switch stars to show any rating you want
Please fix it

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
```
Comment (alexander-ignatow) [inline src/styles/blocks/card.scss:96]: Are you sure you need to hardcode the button's width and height?

File: src/styles/blocks/container.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  padding: 50px 40px 72px;
+}
```
Comment (alexander-ignatow) [inline src/styles/blocks/container.scss:8]: If you are using grid - please implement as it is described in the [task](https://github.com/mate-academy/layout_catalog/blob/master/grid.md)

File: src/styles/normalize.scss [hunk 4]
```diff
@@ -0,0 +1,6 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```
Comment (alexander-ignatow) [inline src/styles/normalize.scss:5]: If you are setting box-sizing do it using *, because it is _not_ inhereted

File: src/index.html [hunk 5]
```diff
@@ -4,9 +4,391 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
```
Comment (alexander-ignatow) [inline src/index.html:9]: Do you really need all these fonts? Please import only the ones you are using

File: src/styles/blocks/header.scss [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+.header__logo {
+  display: flex;
+  align-items: center;
+}
+
+.header__list-item-link:hover {
+  color: $nav__list-item-link-color;
+}
+
+.header__list-item-link.is-active::after {
```
Comment (Moroz-Dmytro) [inline src/styles/blocks/header.scss:48]: Don't increase specificity if no needed and use nesting with `is-active` class styles

File: src/styles/blocks/header.scss [hunk 7]
```diff
@@ -0,0 +1,57 @@
+.header {
```
Comment (Moroz-Dmytro) [inline src/styles/blocks/header.scss:1]: Use nesting for all styles in this file

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Have you deployed your changes?
I don't see changes regarding this

image

When I hover on "Buy" button content of the page is jumping, please fix this

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    &-count {
+      margin: 16px 16px 0 0;
+    }
+  }
+
+  &__stars {
```
[the remark this answers is not in the document]
Comment (BudnikOleksii) [reply src/styles/blocks/card.scss:52]: Still not fixed. Stars is a separate BEM block, let's move it from the card, in the future, you can reuse it in other places if needed.

File: src/index.html [hunk 9]
```diff
@@ -4,9 +4,384 @@
...
+          <li class="header__list-item">
+            <a class="header__list-item-link" href="#smarthones">SMARTPHONES</a>
+          </li>
+          <li class="header__list-item">
+            <a
+              class="header__list-item-link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+            LAPTOPS & COMPUTERS
```
[the remark this answers is not in the document]
Comment (BudnikOleksii) [reply src/index.html:38]: The formatting is not fixed yet, check the previous comments.

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Great job👍
Just forgot to move stars to a separate block, this block can be reusable.

File: src/index.html [hunk 10]
```diff
@@ -4,9 +4,384 @@
...
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+          </div>
+
+          <p class="card__review-count">Reviews: 5</p>
+
+        </div>
```
[the remark this answers is not in the document]
Comment (roman-mirzoian) [reply src/index.html:89]: @SmartSergiZ Maybe you forgot to fix it?

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Good job, but there is quite a bit of work to be done.
Do not forget to process comments from mentors.
Also, your header is now visually slightly different from the design.
Design:
image
Your app:
image

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**